js/sales.js

```diff
@@ -8,80 +8,80 @@
         
-        for(var i = 0; i < this.simulatedCookiesPerHour.length; i++){
+        for(let i = 0; i < this.simulatedCookiesPerHour.length; i++){
             tableStoreRow.appendChild(createDataCell(this.simulatedCookiesPerHour[i], 'td'));
         }
         document.querySelector('tbody').appendChild(tableStoreRow);
         return tableStoreRow;
     }
 
     CookieStore.prototype.calculateStoreTotal = function(){
-        var total = 0;
-        for(var i = 0; i < this.simulatedCookiesPerHour.length; i++){
+        let total = 0;
+        for(let i = 0; i < this.simulatedCookiesPerHour.length; i++){
             total += this.simulatedCookiesPerHour[i];
         }
         return total;
     }
 
     function createStoreTableContents(){
-        var hoursTableRow = createHoursHeadRow();
+        const hoursTableRow = createHoursHeadRow();
         hoursTableRow.appendChild(createDataCell('Daily Location Total', 'th', 'col'));
 
-        var allStoresTotal = 0;
-        for(var i = 0; i < cookieStoreLocations.length; i++){
-            var store = cookieStoreLocations[i];
-            var storeTableRow = store.render();
-            var total = store.calculateStoreTotal();
+        let allStoresTotal = 0;
+        for(let i = 0; i < cookieStoreLocations.length; i++){
+            const store = cookieStoreLocations[i];
+            const storeTableRow = store.render();
+            const total = store.calculateStoreTotal();
             allStoresTotal += total;
             storeTableRow.appendChild(createDataCell(total, 'td'));
         }
-        var totalsRow = createTotalsFooterRow();
+        const totalsRow = createTotalsFooterRow();
         totalsRow.appendChild(createDataCell(allStoresTotal, 'td'));
     }
 
     function createHoursHeadRow(){
         //use military time
-        var openingHour = 6;
-        var standardTimeOffSet = 12;
+        const openingHour = 6;
+        const standardTimeOffSet = 12;
         
-        var tableRow = document.createElement('tr'); 
+        const tableRow = document.createElement('tr'); 
         tableRow.appendChild(createDataCell('Store Locations', 'th', 'col')).setAttribute('id', 'store-locations');
 
-        for(var i = 0; i < OPEN_HOURS; i++){
-            var am = ':00am';
-            var pm = ':00pm';
-            var textContent;
-            var militaryHour = openingHour + i;
+        for(let i = 0; i < OPEN_HOURS; i++){
+            const am = ':00am';
+            const pm = ':00pm';
+            let textContent;
+            let militaryHour = openingHour + i;
             
             if(militaryHour === 12){
-                textContent = '12' + pm;
+                textContent = `12${pm}`;
             } else if(militaryHour === 24 || militaryHour === 0){
-                textContent = '12' + am;
+                textContent = `12${am}`;
             } else if(militaryHour < 12){
                 //1am-11am
                 textContent = militaryHour + am;
```

Comment:
We could change the assignment of `textContent` on lines 87 and 90 to also use template literals - we're still creating a String via concatenation, so this could also be `` `${militaryHour}${am}` ``. _However_, for something this simple, I think an argument could also be made that it's easier to read with simple concatenation instead of template literals.

I'll leave it up to you if you'd like to change these two lines or not.